GameDataParser/Parsers/ItemParser.cs

```diff
@@ -518,23 +415,21 @@ protected override List<ItemMetadata> Parse()
         return rewards;
     }
 
-    private static void ParseHair(XmlNode slot, ItemMetadata metadata)
+    private static void ParseHair(Slot slot, ItemMetadata metadata)
     {
-        int assetNodeCount = slot.SelectNodes("asset").Count;
-        XmlNode asset = slot.FirstChild;
-
-        XmlNode scaleNode = slot.SelectSingleNode("scale");
+        int assetNodeCount = slot.asset.Count;
+        Slot.Scale scaleNode = slot.scale?.FirstOrDefault();
 
         CoordF defaultCoord = CoordF.Parse("0, 0, 0");
```

Comment:
I'm pretty sure you can replace all usages of this with `default`

Reply:
What exactly? The parse("0, 0, 0")?

Reply:
the whole variable is unnecessary since structs implicitly have a default value of all their bytes being 0. This value can be used by the keyword `default`.